### ISSUE TITLE
Notification that MUC has been closed although it is active

### ISSUE THREAD

**Issue by sebasuh** [iNPUTmice/Conversations#4472]
I opened a private MUC on mailbox.org for myself and a few friends 2 weeks ago. I use Conversations my friends use Quicksy.
After a while they stop getting notifications and when they open the chat they see a toast-like notification at the bottom of the screen telling them the group chat has been closed ("Gruppenchat wurde geschlossen").

I have already searched the issues and did a quick code search but nothing similar came up.

Has this happened to anybody else or does anybody know why this could be happening? I am not that deep into the XMPP rabbit hole to know if it could be an S2S, S2C or exclusively a client problem.

Could it be an issue with the private group chat creation option ("Privaten Gruppenchat erstellen")? Is it non persistent?

**Comment by licaon-kter** [comment]
(As the Readme says, the repo has moved to https://codeberg.org/iNPUTmice/Conversations anyway)

Your server updated its software once in Dec 2018: https://compliance.conversations.im/server/mailbox.org/ maybe inform them that latest ejabberd is 24.02?

 So the MUC works fine for you? They still see your messages? Can try to reinvite them?

**Comment by sebasuh** [comment]
> (As the Readme says, the repo has moved to https://codeberg.org/iNPUTmice/Conversations anyway)
> 
> 
> Your server updated its software once in Dec 2018: https://compliance.conversations.im/server/mailbox.org/ maybe inform them that latest ejabberd is 24.02?
> 
> So the MUC works fine for you? They still see your messages? Can try to reinvite them?


ah you are right, sorry. It was somewhere at the back of my mind when I last browsed the repo a while ago but I jumped straight to the issues and code search and did not remember. I will open an issue at the new location and link to this one for reference if your ideas do not solve the issue. 

Thanks for the ideas and the reminder for the compliance tester.
I already contacted the server support about this issue but their primary focus does not seem to be on the XMPP server if it is that much out of date. Probably not many users on the server, so their engineers focus their energy on the other services they provide
